client/src/pages/LoginForm.css

```diff
@@ -73,3 +66,33 @@ button[type="submit"] {
 	color: red;
 	margin-bottom: 9px;
 }
+
+/* MEDIA QUERIES */
+
+@media (min-width: 600px) {
+	form {
+		margin-top: 50px;
+		max-width: 70%;
+	}
+
+	.login-group {
+		margin-top: 50px;
+		margin-bottom: 20px;
+		max-width: 70%;
+	}
+
+	input {
+		width: 50%;
+	}
+
+	.loginButton {
+		width: 70%;
+		font-size: 20px;
+	}
+}
+
+@media (min-width: 600px) {
+	.login-group {
```

Comment:
I am just curious why you created  `@media (min-width: 600px)` twice.

Reply:
Thanks for catching that error, I will fix it before merging :-)